scripts/serve-pimcore5.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# install apache and bzip PHP extension
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
I don't like the idea of having the "install apache" code now in 2 places. That's not necessarily a blocker for getting this PR merged, just something I want to think about a better solution for